config/settings.yml

```diff
@@ -45,3 +45,7 @@ c2m_sql_limit: 1000
 checksum_algos: ['md5'] # 'sha1' 'sha256'
 
 zip_storage: '/tmp' # override in #{RAILS_ENV}.yml
+
+worker_dashboards:
```

Comment:
would like a comment clarifying what these are for, and/or change this key to be something more descriptive, like `resque_dashboard_hostnames`

Reply:
- I included the commented out config files that are used in shared_configs to wire resque up, to give the reader some added context, but if they're confusing, I'm happy to leave them out. 

- I don't know if we need to limit resque route actions. From a glance at the dashboard, I don't see any "dangerous" looking tabs.